data-manipulation/hashing/murmur/hash-data-using-murmur3.yml

```diff
@@ -28,3 +28,21 @@ rule:
         - and:
           - number: 0x87c37b91114253d5 = 64-bit c1 for 128-bit hash
           - number: 0x4cf5ad432745937f = 64-bit c2 for 128-bit hash
+        - and:
+          - basic block:
+            - and:
+              - mnemonic: rol
+              - number: 15 = r1
+              description: k ROL r1
+          - basic block:
+            - and:
+              - mnemonic: rol
+              - number: 13 = r2
+              description: hash ROL r2
+        - basic block:
+          - and:
+            - count(mnemonic(shr)): 3
+            - number: 16
+            - number: 13
+            description: 'hash >> 16; hash >> 13; hash >> 16'
+

```

Comment:
unrelated to this PR, can you please clean up the top-level `and`/`or`, while you're at it?